plugins/output-redis/src/main/scala/com/stratio/sparkta/plugin/output/redis/RedisOutput.scala

```diff
@@ -57,20 +54,39 @@ class RedisOutput(keyName: String,
    * @param metricOperations that will be saved.
    */
   override def upsert(metricOperations: Iterator[(DimensionValuesTime, Map[String, Option[Any]])]): Unit = {
-    metricOperations.toList.groupBy(upMetricOp =>
-      AggregateOperations.keyString(upMetricOp._1, upMetricOp._1.timeDimension, fixedDimensions))
-      .filter(_._1.size > 0).foreach(collMetricOp => {
-      collMetricOp._2.foreach(metricOp => {
-        val timeDimension = metricOp._1.timeDimension
-        val hashKey = collMetricOp._1 + IdSeparator + metricOp._1.dimensionValues
-          .map(dimVal => List(dimVal.getNameDimension, dimVal.value.toString))
-          .flatMap(_.toSeq).mkString(IdSeparator) + IdSeparator + timeDimension + IdSeparator + metricOp._1.time
+    val dimAggGrouped = filterNonEmptyMetricOperations(groupMetricOperationsByHashKey(metricOperations.toList))
+
+    for {
+      (hashKey, dimensionsAggregations) <- dimAggGrouped
+      (dimensionsTime, aggregations) <- dimensionsAggregations
+      (aggregationName, aggregationValue) <- aggregations
+      currentOperation = operationTypes.get.get(aggregationName).get._1
+    } yield {
+      if (supportedWriteOps.contains(currentOperation)) hset(hashKey, aggregationName, aggregationValue.get)
+      else log.warn(s"Operation not supported in cube: $aggregationName")
```

Comment:
Put the operation not supported in the log: log.warn(s"Operation $currentOperation not supported in the cube with name $aggregationName")